build.sh

```diff
@@ -16,7 +16,7 @@ usage()
     echo
     echo "    CONFIG is an optional configuration preset for a specific SoM or board."
     echo "    Available configs:"
-    echo "        - k26           Xilinx Kria K26 SoM"
+    echo "        - kria           Xilinx Kria K26/K24 SoMs"
```

Comment:
As below, we should keep backwards compatibility here, so pehaps:

```
- - k26           Xilinx Kria K26 SoM
+ - k26           Same as "kria" (deprecated)
- - kria           Xilinx Kria K26/K24 SoM
```

Reply:
Yes, good idea to mark the k26 options as "deprecated" in the help.  This way, we could consider possibly removing the old option in the future, should that ever make sense.